courses/urls.py

```diff
@@ -11,6 +11,7 @@
 router.register(r"course_runs", views.CourseRunViewSet, basename="course_runs_api")
 
 urlpatterns = [
+    re_path(r"^api/v1/", include(router.urls)),
```

Comment:
Do we need to define any default versioning scheme ? 
For future corresponding, what value be expected in request.version state here ?

Reply:
Views moved to `views/v1/` for better code management